### PR TITLE
chore: update a few dependencies (thiserror, subtle, base64*)

### PR DESCRIPTION
**Pull request by Stebalien**
Unfortunately we can't yet update the sha2 or hkdf until https://github.com/zkcrypto/bls12_381/issues/139 is resolved.

### CODE REVIEW

**Comment by Stebalien** [comment]
Based on #76 (to make CI work).

File: src/signature.rs [hunk 1]
```diff
@@ -306,7 +306,7 @@ pub fn verify_messages(
 mod tests {
     use super::*;
 
-    use base64::STANDARD;
+    use base64::prelude::*;
```
Comment (vmx) [inline src/signature.rs:309]: I personally prefer not having glob imports. Here's only for tests, which could be OK, but importing the concrete things should be easy enough :)

Comment (Stebalien) [reply src/signature.rs:309]: Yeah, I can do that.

**Review by vmx — APPROVED** [review]
Thanks!

**Pull request merged**